hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -82,6 +82,7 @@ cdef class PureMarketMakingStrategy(StrategyBase):
                  bid_order_optimization_depth: Decimal = s_decimal_zero,
                  add_transaction_costs_to_orders: bool = False,
                  asset_price_delegate: AssetPriceDelegate = None,
+                 price_is_mid: bool = True,
```

Comment:
This can be the new Enum described in https://github.com/CoinAlpha/hummingbot/issues/2036